### ISSUE TITLE
Update icon: Create (Write)

### ISSUE THREAD

**Issue by folletto** [Automattic/gridicons#242]
![screen shot 2017-07-28 at 15 38 54](https://user-images.githubusercontent.com/4389/28724131-f2114c7a-73b0-11e7-92f1-e333cba66f09.png)

![screen shot 2017-07-28 at 15 37 20](https://user-images.githubusercontent.com/4389/28724137-f5ac1978-73b0-11e7-8954-4d3aab200f60.png)

We had some feedback on mobile that the "Write" icon (`gridicon-create`) is a bit too obscure. This might be due to various reason, but we could also take the chance to resolve the "border inside border" effect we currently have, as well as creating something more distinctive (ref: Twitter).

`gridicon-create`:

![screen shot 2017-07-28 at 16 20 49](https://user-images.githubusercontent.com/4389/28724149-fee2f016-73b0-11e7-809a-07be7976148b.png)

For reference, before we used `gridicon-pencil`, which we use now for generic edit:

![screen shot 2017-07-28 at 16 20 30](https://user-images.githubusercontent.com/4389/28724142-f8cd93e8-73b0-11e7-9eba-78f7b7c3d125.png)

Ideally we'd like something clearer as well as more unique.

**Comment by drw158** [comment]
Previous work: #84

**Comment by drw158** [comment]
Just some thoughts:

It's hard to imagine something more clear for 'Write' than the pencil icon. The 'pencil-in-a-box' icon is also pretty standard from my POV, but I realize I'm biased in that way. It's possible that the Write icon is not even clear that it's a pencil, which would be a problem! I like the idea of just a pencil. Some ideas I would try:

- make it more clear that it's a pencil
- try to make the 'box' look like paper
- try a pencil icon that's drawing a line

Some icon ideas I like:

<img width="164" alt="screen shot 2017-07-28 at 10 50 06 am" src="https://user-images.githubusercontent.com/618551/28725578-7ca8d26e-7383-11e7-904e-ae8d1cf3c746.png">
<img width="152" alt="screen shot 2017-07-28 at 10 49 51 am" src="https://user-images.githubusercontent.com/618551/28725579-7caeab58-7383-11e7-94e8-156ab9252b14.png">
<img width="28" alt="screen shot 2017-07-28 at 10 49 28 am" src="https://user-images.githubusercontent.com/618551/28725580-7cb29b3c-7383-11e7-810d-f30b17505c71.png">
<img width="154" alt="screen shot 2017-07-28 at 10 47 13 am" src="https://user-images.githubusercontent.com/618551/28725581-7cb482bc-7383-11e7-9f81-ab6da99ef560.png">
<img width="145" alt="screen shot 2017-07-28 at 10 46 57 am" src="https://user-images.githubusercontent.com/618551/28725582-7cba90da-7383-11e7-9a78-a5356c305a6e.png">

**Comment by folletto** [comment]
Yay thanks! I think there are in there already a few explorations that can be done and iterated upon!

**Comment by joanrho** [comment]
The Noun Project has a great collection of creative community-sourced, but moderated/curated icons. Here's what they have for [Create](https://thenounproject.com/search/?q=create) and [Write](https://thenounproject.com/search/?q=write). In addition to the standard sheet of paper and superimposed pencil icon, there are other options like the plus sign in a circle (+), or a plus sign over a sheet of paper. It's pretty standard to see the solo pencil used in instances where Editing is the function (see noun project [Edit](https://thenounproject.com/search/?q=edit)).

Since not all posts will include writing (e.g., some may include just image assets), how about a direction that explores the (+) sign instead of a pencil?

**Comment by drw158** [comment]
I think the problem with going more abstract or generic (like just a plus icon) is that, in trying to accommodate and include everything (images, text, video, etc), you lose the clarity and instant recognition that comes with an icon that is more distinguishable.

**Comment by folletto** [comment]
I agree on the plus... unless we are able to evolve it in a way that feels still specific enough? I just don't want to preclude too much in the initial brainstorming phase ;) 

I like all the other ideas thrown in. Especially doing some benchmarking, which will also help avoiding to be too generic, while being still clear. :)

**Comment by folletto** [comment]
Here's some explorations:

![screen shot 2017-09-05 at 00 15 05](https://user-images.githubusercontent.com/4389/30039909-4cdbce8c-91cf-11e7-98f0-fd4d2c4dd140.png)

**Comment by folletto** [comment]
This size is kinda tricky for the limitation in the amount of details we can have...

![artboard](https://user-images.githubusercontent.com/4389/30056747-d7507c0a-922b-11e7-82db-61ff09e348e1.png)

**Comment by jameskoster** [comment]
> Ideally we'd like something clearer as well as more unique.

I think this will be very challenging and we should probably try to be one or the other.

For clarity I think it will be difficult to beat a pen/pencil. I really like this one:

![pencil](https://user-images.githubusercontent.com/618551/28725578-7ca8d26e-7383-11e7-904e-ae8d1cf3c746.png)

But it's essentially a riff on what we already have. If that's already too obscure this one probably would be as well? Maybe just a page with a plus icon (number 11 in your sketches) would be clearer.

If we want something unique, I really like the Quill/Nib idea(s). They have character and hint at the aspirational craft of physical writing (Calligraphy).

**Comment by folletto** [comment]
That icon has the same "square in square" effect that made me open this issue... it feels odd in the toolbar, but even worse on mobile when it's literally inside a square. It's just coincidental... but it feels odd.

I really like the quill, I'm just worried Twitter already made it "its" thing... and it's SO on brand for them...

> I think this will be very challenging and we should probably try to be one or the other.

We're a smart bunch, I'm sure we can :D

**Comment by jameskoster** [comment]
> I like particularly the direction (2) is going: high precision pen/pencil, with content.

That pencil looks like it's writing RTL, could we have an LTR version and display the relevant icon based on your locale? xD

In all seriousness, my concern with this icon is that the additional detail on the pen/pencil actually makes it less clear. I can now see a syringe/epipen, or a jackhammer. The original pencil was un-mistakably a pencil, to my eye. I wonder how that would look with an underscore, or an applied '+' like `gridicons-add-image`.

There are only so many metaphors for writing in the physical world and most of them have been used. You're right, the quill is probably too Twitter'y. I wonder if there's a digital icon we could use? Like the text cursor?

![text-cursor](https://cldup.com/apseGGFJwV.png)

**Comment by folletto** [comment]
> The original pencil was un-mistakably a pencil, to my eye.

Or a stick. Or a chisel. Or a pipette. .... I think that's mostly habit. It's hard to do this kind of change. :)

Icons, especially at this size, will always have some degree of uncertainty – in general, they aren't clear at all, as we should label them. :D 

> I wonder how that would look with an underscore, or an applied '+' like gridicons-add-image.

Good point. Plus as Joan mentioned above has a strong connotation.

> Like the text cursor?

Yesssss! More divergent ideas! :D

**Comment by folletto** [comment]
5
![5](https://user-images.githubusercontent.com/4389/30174053-a89d9786-93f1-11e7-872d-593ca9372908.png)

Trying to make it look less needle-y and adding the plus for additional clarity.

**Comment by jameskoster** [comment]
I think that's definitely an improvement :)

**Comment by jasmussen** [comment]
> I think the problem with going more abstract or generic (like just a plus icon) is that, in trying to accommodate and include everything (images, text, video, etc), you lose the clarity and instant recognition that comes with an icon that is more distinguishable.

I think this touches at the core of it. 

That said, I do think a simple plus could work, as it's becoming a standard for "new stuff" across a wide variety of places. It may not read as well as Davide's last pencil there, but another benefit it has is that it can look really nice in context of the other "big five". Regardless of which implementation we go with, the icon should be designed next to these other four ones:

![screen shot 2017-09-27 at 15 54 30](https://user-images.githubusercontent.com/1204802/30917396-312237e2-a39c-11e7-8c6c-1576432e330a.png)

In that vein I agree with James <a href="https://github.com/Automattic/gridicons/issues/242#issuecomment-327138775">here</a>, it should either be a super simple pencil and paper shape like that, or we should consider whether the plus could work.

If someone has an abundance of energy we could even run an A/B test with a bunch of icons, even the plus.

**Comment by folletto** [comment]
Note: we had done tests on mobile and the ”simple" icon wasn't effective, so if we change it, I would try something _else_ than an abstract pen. Which yes, a simple "+" could do. :)

**Comment by mattmiklic** [comment]
I'm a little late to the party, but I think one thing that makes the current icon unclear is the perspective and size difference between the page and the pencil above it. If you compare it to the [iOS compose icon](https://developer.apple.com/documentation/uikit/uibarbuttonsystemitem/1617156-compose) that (I think) inspired it, the position of the pencil on the page, its hollow shape, and its size all help. But even then, it's so abstract that you kind of have to know what it means from the context where it appears -- like in the toolbar of the Mail app.

I like the idea of using the plus symbol here because our editor isn't just used for writing; people can also use it to publish photo galleries, movies, music or podcasts.

**Comment by folletto** [comment]
Ok, so I did a couple of examples with the + we have, and a couple of variations just to avoid leaving any stone unturned as unlikely as it might be:

6
![6](https://user-images.githubusercontent.com/4389/31438069-7e02b148-ae7f-11e7-8ff8-f8c2bd59cd14.png)

7
![7](https://user-images.githubusercontent.com/4389/31438080-804345b2-ae7f-11e7-9e3b-4fc8fbe64609.png)

8
![8](https://user-images.githubusercontent.com/4389/31438098-82597f7e-ae7f-11e7-990a-394fea781c47.png)

9
![9](https://user-images.githubusercontent.com/4389/31438115-84399356-ae7f-11e7-880c-0389dcacba90.png)

---

I think the simple, existing "+" does probably a better job in pretty much any scenario, and would back up @joanrho's original point of being open to any kind of content.

If we like the "pure plus" direction (7), then the remaining question is sematic: shall we still replace the existing `cridicons-create` one with the same (or similar) plus, or simply start using the existing `gridicons-plus`?

**Comment by jasmussen** [comment]
I like 7, even though for sure it will take some getting used to. It could be worth a test, maybe?

**Comment by mattmiklic** [comment]
I'm kind of drawn to 8, it seems to best convey "add a new thing." I may like it because it seems familiar, since Snapchat uses a similarly-styled icon to add a new story item:

![img_3275](https://user-images.githubusercontent.com/1396552/31548875-61cfacb6-affa-11e7-80cc-e4dee1a43860.jpg)

Instagram goes with something pretty close to a pure + though, just with a roundrect around it — this might be similar to how we would treat a pure + on mobile (though having room to include the "Write" label wouldn't be a bad idea).

![img_3276](https://user-images.githubusercontent.com/1396552/31549428-291d1b54-affc-11e7-8550-04a9c6d620b2.jpg)

**Comment by jasmussen** [comment]
I can dig 8, with a few tweaks. There's an optical line imbalance with the word next to it here:

![screen shot 2017-10-13 at 15 59 35](https://user-images.githubusercontent.com/1204802/31549806-8efd11ee-b02f-11e7-9775-7004063c09d4.png)

This can either be fixed by moving the plus slightly more inside the box like in the snapchat icon, or potentially moving the entire icon up a bit. Seems worth experimenting with!

**Comment by folletto** [comment]
It still bothers me a little it's a "box-in-a-box", but I guess it can't be avoided that much... it's semantically probably more effective than the pure "+"... so maybe we can review the UI later to avoid it?

Anyhow... here two iteration on 8 (one max size, one smaller)

10
![10](https://user-images.githubusercontent.com/4389/31550044-eb6ce8e4-b027-11e7-9159-6a59fe2c37e1.png)

11
![11](https://user-images.githubusercontent.com/4389/31550049-edef77c6-b027-11e7-8827-16900dee5575.png)

**Comment by jasmussen** [comment]
I could 👍 👍  :shipit:  to 10. It's visually nice, deliciously simple, and the plus is the clearest of the two. It also aligns better in the button.

**Comment by drw158** [comment]
Looks good and 10/11 are shippable, but I still stand by my original stance that if it becomes generic we lose that instant recognition. I think WordPress in general is about writing text primarily, and media is secondary or supportive. For example, if I publish a photo, the platform is designed for text to accompany that photo (nothing is stopping you from just posting a photo of course). This could be narrow thinking or might not align with company goals.

--
Dave Whitley / Designer at Automattic

**Comment by folletto** [comment]
> I think WordPress in general is about writing text primarily, and media is secondary or supportive. 

I'm with you here in a general sense, but as mentioned in the original issue at the top we did tests and the current icon doesn't perform well. With the plus it might get some extra "meaning of add" that is lost in the current one so we have a chance it could perform better. 

I personally would have liked a more "branded" approach like Twitter (they aced this, with a little bit of luck, sure) but lacking that, I'd still try to fix the clarity part of the equation.

tl;dr: even if generics/media is one of the rationales mentioned, and it's valid, it's not the primary reason for this change.

**Comment by drw158** [comment]
Closing since a new icon was made in #256  for this purpose instead of updating the Write icon.
